ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -146,7 +146,9 @@
     "AccessControl": [
       {
         "UserRoles": [
-          "caseworker-civil-solicitor"
+          "[CREATOR]",
```

Comment:
why `caseworker-civil-solicitor` role is removed?

Reply:
When respondent solicitor 2 logs in and runs `Respond to Claim`
we dont want to show them the DOB / Address field for Respondnet1.

Therefore these fields need to be behind their own case roles, so Respondent 1 only sees what is applicable to them. Same for Respondent2 field. In a same soliictor scenario, the user will have bith case roles and so will see both fields